framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1275,15 +1274,8 @@ protected void addComputedTypeAnnotations(
         defaults.annotate(tree, type);
 
         if (iUseFlow) {
-            Value as;
-            if (tree.getKind() == Kind.POSTFIX_DECREMENT
-                    || tree.getKind() == Kind.POSTFIX_INCREMENT) {
-                // Dataflow incorrectly treats postfix as prefix.
-                // See Issue 867: https://github.com/typetools/checker-framework/issues/867
-                as = getInferredValueFor(((UnaryTree) tree).getExpression());
```

Comment:
@bitterfox I think you mis-interpreted @smillst comment. I think it was nicer to have this logic in the GenericATF instead of moving it to the BaseTypeVisitor. The visitors shouldn't modify the types.
The suggestion was to just update the comment in BaseTypeVisitor.